required/offline_API_wrapper.js

```diff
@@ -68,7 +81,18 @@ var API = {
 	LMSStore: function(force) {
 		if (window.ISCOOKIELMS === false) return;
 		if (!force && API.cookie("_spoor") === undefined) return;
-		API.cookie("_spoor", JSON.stringify(this.data));
+
+		var stringified = JSON.stringify(this.data);
+
+		API.cookie("_spoor", stringified);
+
+		// a length mismatch will most likely indicate cookie storage limit exceeded
+		if (API.cookie("_spoor").length != stringified.length) {
```

Comment:
I know the white space is, well, missing, but this was done in the spirit of `createResetButton`. That's no great justification I know.